base_partner_sequence/views/partner_view.xml

```diff
@@ -7,8 +7,8 @@
         <field name="arch" type="xml">
             <field name="ref" position="attributes">
                 <attribute
-                    name="attrs"
-                >{'readonly': [('is_company', '=', False), ('parent_id', '!=', False)]}</attribute>
+                    name="options"
+                >{'readonly': [['is_company', '=', False], ['parent_id', '!=', False]]}</attribute>
```

Comment:
Attr options does not work from my side. my suggestion:
`<attribute name="readonly">not is_company and parent_id</attribute>`